src/components/SelectMateria.js

```diff
@@ -72,14 +86,79 @@ const SelectMateria = ({ materiasToShow }) => {
           placeholder="Buscar Materia..."
           _placeholder={{ color: "gray.200" }}
         />
-        <InputRightElement children={<SearchIcon color="primary.500" />} />
+        <InputRightElement
+          children={
+            <Menu closeOnSelect={false}>
+              <MenuButton
+                colorScheme="primary"
+                variant="outline"
+                border="none"
+                as={Button}
+                rightIcon={
+                  <Tooltip placement="top" label="Filtrar materias">
```

Comment:
```suggestion
                  <Tooltip placement="top" label="Filtrar materias por día">
```

Al menos hasta que tengamos mas filtros?